### DISCUSSION TITLE
Query for tag field search

### DISCUSSION THREAD

**Discussion by ludvig-sandstrom** [xetorthio/jedis#3066] [Q&A] [answered]
Hi!
I'm trying to execute a tag field search with ftSearch, but I fail to understand how to implement it correctly. I tried to look up examples, but failed to find one. Could anyone provide a simple example?
I know the pure redis syntax and have tried to translate that into a query ( FT.SEARCH [index-name] "tag:{value}" ) but have so far failed miserably. I guess that the querystring should still be the value, and suspect the tag should be defined in a filter (addFilter) or similar, but none of my attempts have so far worked out.
Really grateful for any assistance.

**Comment by sazzad16** [comment, marked as answer]
Hopefully following tests would help.

https://github.com/redis/jedis/blob/e92df8220597e8b05175ae9ea9d7452b9d2a8a6d/src/test/java/redis/clients/jedis/modules/search/SearchTest.java#L1058-L1158 

Perhaps you need to do `@tag:{value}` with an `@`.

**Reply by ludvig-sandstrom** [reply]
It was worse than that....built the tag value and managed to get an empty string to build it from in the first of my tests....so all stupidity....
Thanks again!

**Comment by ludvig-sandstrom** [comment]
Thanks a lot! Probably should've been able to guess that, but you made it a lot easier!
